CMakeLists.txt

```
@@ -246,6 +251,13 @@ if("${DDC_BUILD_KERNELS_SPLINES}")
   find_package(Ginkgo 1.7.0 EXACT REQUIRED)
   target_link_libraries(DDC INTERFACE Ginkgo::ginkgo)
   target_compile_definitions(DDC INTERFACE ginkgo_AVAIL)
+
+  # Lapack
+  find_package(LAPACK REQUIRED)
+  find_package(LAPACKE REQUIRED)
+  target_link_libraries(DDC INTERFACE LAPACK::LAPACK)
+  # The following should not be necessary for LAPACKE >=3.9.1
+  # target_compile_definitions(DDC INTERFACE HAVE_LAPACK_CONFIG_H LAPACK_COMPLEX_CPP)
```

```suggestion
  # Lapack
  find_package(LAPACKE REQUIRED)
  target_link_libraries(DDC INTERFACE ${LAPACKE_LIBRARIES})
  target_include_directories(DDC INTERFACE ${LAPACKE_INCLUDE_DIRS})
```

I think FindLAPACKE is meant to be used like this.